lib/HTFeed/Stage.pm

```diff
@@ -23,11 +23,14 @@ sub new {
         croak __PACKAGE__ . ' can only construct subclass objects';
     }
 
+    # No class inheriting from HTFeed::Stage needs to instantiate
+    # its own $self->{job_metrics} for storing JobMetrics.
     my $self = {
         volume => undef,
         @_,
         has_run => 0,
         failed  => 0,
+	job_metrics => HTFeed::JobMetrics->get_instance
```

Comment:
This makes it so that each object that inherits from `Stage` should be able to access `$self->{job_metrics}`, so we don't need to say `use HTFeed::JobMetrics` in every `Stage`-descendant.